ambari-server/src/main/java/org/apache/ambari/server/controller/AmbariManagementControllerImpl.java

```diff
@@ -6236,7 +6232,7 @@ private Boolean getUnlimitedKeyJCERequirement(ComponentInfo componentInfo, Secur
     String notManagedHdfsPathList = gson.toJson(notManagedHdfsPathSet);
     clusterLevelParams.put(NOT_MANAGED_HDFS_PATH_LIST, notManagedHdfsPathList);
 
-    StackInfo stackInfo = ambariMetaInfo.getStack(stackId.getStackName(), stackId.getStackVersion());
+    ambariMetaInfo.getStack(stackId.getStackName(), stackId.getStackVersion());
```

Comment:
same, no idea why this is needed. Originally i supposed that this is some kind of check (coz of internal implementation), but seems this left from old code